modules/nicknames/renameMember.js

```diff
@@ -0,0 +1,56 @@
+const {truncate} = require("../../src/functions/helpers");
+
+renameMember = async function (client, guildMember) {
+    const roles = client.configurations['nicknames']['config'];
+    const moduleModel = client.models['nicknames']['User'];
+
+    let rolePrefix = '';
+    let userRoles = guildMember.roles.cache.sort((a, b) => a.position - b.position).map(r => r.id);
```

Comment:
```suggestion
    let userRoles = guildMember.roles.cache.sort((a, b) => b.position - a.position).map(r => r.id);
```
Actually, sorting is wrong sorry about this